src/Serializer/ArticleVoRNormalizer.php

```diff
@@ -101,7 +101,7 @@ protected function denormalizeArticle(
                     new ArraySequence(array_map(function (array $block) use ($format, $context) {
                         return $this->denormalizer->denormalize($block, Block::class, $format, $context);
                     }, $authorResponse['content'])),
-                    $authorResponse['doi'],
+                    $authorResponse['doi'] ?? null,
```

Comment:
Currently nothing testing this hence previous passing build.

Makes me think that the normalizer tests should take the samples from `elife/api` and denormalize then normalize them, and make sure the result is the same.